.github/DEVELOPMENT.md

```diff
@@ -1,10 +1,11 @@
 # Development
 
-After [forking the repository on GitHub](https://docs.github.com/en/get-started/quickstart/fork-a-repo), install required packages with [Yarn](https://yarnpkg.com):
+After [forking the repository on GitHub](https://docs.github.com/en/get-started/quickstart/fork-a-repo), install required packages with [Yarn](https://yarnpkg.com) and set the correct Node version with [nvm](https://github.com/nvm-sh/nvm#installing-and-updating):
```

Comment:
Hmm, I don't love telling everybody that they have to specifically use nvm in order to set up this repository. Is there a phrasing we can use that suggests this without outright requiring it?

Maybe adding below the shell script something like this?

```md
> Tip: Consider using [nvm](https://github.com/nvm-sh/nvm#installing-and-updating) and running `nvm install` to sync the Node version used to develop on this repository.
```

Reply:
I suppose I'm [0/5](https://handbook.mattermost.com/company/about-mattermost/list-of-terms?q=terms#0-5-1-5-2-5-3-5-4-5-5-5) as long as it's documented. I'll make the suggested change.

Reply:
Aside: I like that term a lot... and _really_ like the public docs 😄 very nice.